### PR TITLE
W4a hackathon

### PR DESCRIPTION
**Pull request by zorkow**
Everything accessible with exception of correct answer if original is wrong.

### CODE REVIEW

File: js/game-page.js [hunk 1]
```diff
@@ -257,11 +259,31 @@ H5P.ArithmeticQuiz.GamePage = (function ($, UI, ArithmeticType, QuestionsGenerat
...
+      node.parentNode.removeChild(node);
+    };
+
+    this.announce = function() {
+      self.$liveRegion = $('<div>', {
+        'class': 'h5p-baq-live-feedback',
+        'aria-live': 'assertive',
+        'width': '1px',
+        'height': '1px',
+        text: self.correct ? 'correct' : 'incorrect'
```
Comment (thomasmars) [inline js/game-page.js]: strings should be translatable, this can be done through placing them in semantics.json

File: js/game-page.js [hunk 2]
```diff
@@ -218,6 +219,7 @@ H5P.ArithmeticQuiz.GamePage = (function ($, UI, ArithmeticType, QuestionsGenerat
 
     this.$scoreWidget = $('<div>', {
       'class': 'h5p-baq-score-widget',
+      'tabindex': '0',
```
Comment (thomasmars) [inline js/game-page.js]: I don't think we need tabindex for score and several of the other elements that has been given a tabindex, there is no action here so keyboard users does not need to access it.
Readspeakers should be able to access the DOM element even if it doesn't have a tabindex.

**Comment by thomasmars** [comment]
Good stuff zorkow, I have left two comments on the commits, let me know what you think.

**Comment by thomasmars** [comment]
I merged in and made some small changes on the commented parts: https://github.com/h5p/h5p-arithmetic-quiz/commit/de16edea470421f8880e83c40e05f1bc4c78f130
Let me know if you disagree :)

**Comment by zorkow** [comment]
Thomas, sorry I've been snowed under this month. Thanks for the merge.

As for the tabindex on score: it depends on what you want to allow your screen reader user:
be able to lookup their score if and when they like or only have it read out on slide initialisation.

**Pull request merged**